plugins/SiegeWar/config.yml

```diff
@@ -319,12 +319,12 @@ war:
         # If this setting is true, then if a player from the banner controlling side dies,
         # the death points are increased by a certain percentage.
         # TIP: This feature delivers extra balance to battles in which one side is outnumbered by the other.
-        enabled: 'false'
+        enabled: 'true'
```

Comment:
This would increase capping warfare and alt capping. Sure it is a low amount but I'd rather keep it disabled